crates/oxc_linter/src/disable_directives.rs

```diff
@@ -604,3 +693,169 @@ semi*/
             .test();
     }
 }
+
+#[cfg(test)]
+mod test_unused_directives {
+    use oxc_allocator::Allocator;
+    use oxc_ast::Comment;
+    use oxc_parser::Parser;
+    use oxc_semantic::{Semantic, SemanticBuilder};
+    use oxc_span::SourceType;
+
+    use super::{DisableDirectivesBuilder, DisabledRule};
+
+    fn process_source<'a>(allocator: &'a Allocator, source_text: &'a str) -> Semantic<'a> {
+        let source_type = SourceType::default();
+        let parser_ret = Parser::new(allocator, source_text, source_type).parse();
+        let semantic_ret = SemanticBuilder::new().build(allocator.alloc(parser_ret.program));
+        semantic_ret.semantic
+    }
+
+    #[test]
+    fn unused_enable_all() {
+        let allocator = Allocator::default();
+        let semantic = process_source(
+            &allocator,
+            r"
+            console.log();
+            /* eslint-enable */
```

Comment:
Can you modify these tests to also use `oxlint-enable-*` and `oxlint-disable-*` as well, to ensure we have coverage there too?